common/persistence/visibility_dual_manager_test.go

```diff
@@ -582,3 +582,647 @@ func TestDualDeleteUninitializedWorkflowExecution(t *testing.T) {
 		})
 	}
 }
+
+func TestDualListOpenWorkflowExecutions(t *testing.T) {
+	request := &ListWorkflowExecutionsRequest{
+		Domain: "test-domain",
+	}
+
+	ctrl := gomock.NewController(t)
```

Comment:
create this per test case which will take the inner `t` and check validations per test case

Reply:
same in all other tests

Reply:
solved offline